core/src/test/scala/unit/kafka/log/LogLoaderTest.scala

```diff
@@ -134,10 +154,29 @@ class LogLoaderTest {
       }
     }
 
+    def initializeLogManagerForSimulatingErrorTest(hasError: Boolean = false,
+                                                 errorType: ErrorTypes.Errors = null,
+                                                 logDirFailureChannel: LogDirFailureChannel = new LogDirFailureChannel(logDirs.size)
+                                                ): (LogManager, Executable) = {
+      simulateError.hasError = hasError
```

Comment:
We are changing simulateError after initializing LogManager. Perhaps it's clearer to just do that for the first test and avoid passing in hasError and errorType to initializeLogManagerForSimulatingErrorTest()?